src/pages/Home/Homepage.js

```diff
@@ -15,24 +15,24 @@ export function HomePage() {
   const cachingCategory = useSelector(state => state.movies.moviesData);
   const [buttonValue, setButtonValue] = useState('discover');
   const [movies, setMovies] = useState([]);
-  const [trigger, moviesData] = useLazyMovieFetchQuery();
+  const [trigger] = useLazyMovieFetchQuery();
   const dispatch = useDispatch();
 
   useEffect(() => {
     (async () => {
```

Comment:
Instead of having a function that returns another async function and then call it, can we just pass an async function to the hook? Something like this
```suggestion
  useEffect(async () => {
```

Reply:
this is what happens 
![image](https://user-images.githubusercontent.com/81062792/163834766-5eca6632-18f8-4be6-ae72-a05755259547.png)
https://github.com/facebook/react/issues/14920

Reply:
Makes sense